package/gluon-web-logging/luasrc/lib/gluon/web/model/admin/logging.lua

```diff
@@ -0,0 +1,49 @@
+local uci = require('simple-uci').cursor()
+local system = uci:get_first('system', 'system')
+
+local f = Form(translate('Logging'), translate(
+	'If you want to use a remote syslog server, you can set it up here. '
+	.. 'Please keep in mind, that the data is transmitted unencrypted.'
+))
+local s = f:section(Section)
+
+local enable = s:option(Flag, 'log_remote', translate('Enable'))
+enable.default = uci:get_bool('system', system, 'log_remote')
+enable.optional = true
```

Comment:
Setting *optional* on a *Flag* doesn't make sense, as it is always true or false, but never undefined.